chia/simulator/setup_services.py

```diff
@@ -150,7 +152,9 @@ async def setup_full_node(
     overrides = service_config["network_overrides"]["constants"][service_config["selected_network"]]
     updated_constants = replace_str_to_bytes(consensus_constants, **overrides)
     local_bt.change_config(config)
-    override_capabilities = None if disable_capabilities is None else get_capabilities(disable_capabilities)
+    override_capabilities = (
+        None if disable_capabilities is None else get_capabilities(NodeType.FULL_NODE, disable_capabilities)
```

Comment:
The whole list is overridden, so it can't just return the diff without refactoring a bunch of things out of scope to this PR. I refactored the function though, let me know if that's better